### PR TITLE
Fix #194738 gazette.com

### LINKED ISSUE #194738: gazette.com
### Issue URL (Trackers)
 [https://gazette.com/sports/mark-kiszla/kiszla-broncos-win-despite-nix-mistakes/article_00e471a8-d335-5773-a656-0e9c765da68c.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fgazette.com%2Fsports%2Fmark-kiszla%2Fkiszla-broncos-win-despite-nix-mistakes%2Farticle_00e471a8-d335-5773-a656-0e9c765da68c.html)
Username: `@xglossyntx`

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard DNS
DNS setup: | AdGuard Home
DNS server: | Private AdGuard DNS server
Browser: | Firefox
Blocklists: | <b>General:</b><br/>AdGuard DNS

### PR DESCRIPTION
**Pull request by xglossyntx**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

Fix #194738

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by xglossyntx** [comment]
The reason I put them all in this pull request is because they are all owned by Clarity Media Group.

**Comment by xglossyntx** [comment]
I believe that's the last of the annoyances.

**Comment by xglossyntx** [comment]
This pull request turned into a bit of a mess, but it should be ready to merge.

File: AnnoyancesFilter/Popups/sections/subscriptions_specific.txt [hunk 1]
```diff
@@ -54,6 +54,7 @@ armidaleexpress.com.au,bendigoadvertiser.com.au,canberratimes.com.au,centralwest
 !
 ! SECTION: Subscriptions - Regular rules
 !
+/cosprings/loader.min.js$domain=coloradopolitics.com|denvergazette.com
```
Comment (Alex-302) [inline AnnoyancesFilter/Popups/sections/subscriptions_specific.txt:57]:
```suggestion
||loader-cdn.azureedge.net/prod/cosprings/loader.min.js$domain=coloradopolitics.com|denvergazette.com
```
more obvious

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
@xglossyntx Hi! Is `x**l**k@protonmail.com` your email?)

**Comment by xglossyntx** [comment]
> @xglossyntx Hi! Is `x**l**n*@protonmail.com` your email?) I want to say something.

Yes

**Pull request merged**